pytorch_toolkit/ote/ote/modules/trainers/mmdetection.py

```diff
@@ -69,6 +69,6 @@ def _update_configuration_file(self, config_path, update_config):
             classes = [category_dict['name'] for category_dict in categories]
         update_config_dict = self.classes_list_to_update_config_dict(cfg, classes)
         cfg.merge_from_dict(update_config_dict)
-        cfg.dump(config_path)
+        cfg.dump(config_path) # TODO(lbeynens): check this, it may be dangerous
```

Comment:
have you checked or how are you going to check?

Reply:
I really think that it is a dangerous operation -- change of config file during training.
Each such change may cause unpredictable issues if one config file is used for several steps (train -> eval -> compress) 
I will fix it in one of my next PR-s.

Reply:
BTW, Ilya, why you do not use `update_config` here? IMHO, it should be considered.

Reply:
We agreed on passing custom args to train.py only. It is ok to read classes from checkpoint and pass them as part of update_config, the only problem is test.py in case of IR (no checkpoint, config is not actual).

This trainer is for custom object detection models only. Other scripts read this configuration and everything should be ok.